### ISSUE TITLE
Rating Slider Example: Testability of value text

### ISSUE THREAD

**Issue by jscholes** [w3c/aria-practices#3259]
The [Rating Slider Example](https://www.w3.org/WAI/ARIA/apg/patterns/slider/examples/slider-rating/) uses `aria-valuetext` in multiple ways:
1. For `aria-valuenow="0"` (i.e. no value set), the value text reads: "Choose a rating from one to ten where 10 is extremely satisfied".  In this case, the value text is essentially acting as user instructions, and is the same regardless of whether the slider has focus or not.
2. For `aria-valuenow="1"` while the slider has focus: "one, extremely dissatisfied".
3. For `aria-valuenow="1"` while the slider does not have focus: "one out of 10, extremely dissatisfied".
4. For `aria-valuenow="10"` while the slider has focus: "ten, extremely satisfied".
5. For `aria-valuenow="10"` while the slider does not have focus: "one out of 10, extremely dissatisfied".
6. For all other `aria-valuenow` values from 2 through 10 while the slider has focus, the number corresponding to the numeric value is spelled out as a word.
	* E.g. when `aria-valuenow` is set to `"5"`, `aria-valuetext` is set to the word "five" (F I V E).
7. For all other `aria-valuenow` values from 2 through 10 while the slider does not have focus, the number corresponding to the numeric value is spelled out as a word, followed by the text " out of ten where ten is extremely satisfied".
	* E.g. when `aria-valuenow` is set to `"5"`, `aria-valuetext` is set to the string "five  out of ten where ten is extremely satisfied".

Item six on this list is causing some testability  concerns within the ARIA-AT project.  Specifically:
* Without manually spelling out the text being sent to the screen reader, it will be difficult for humans to assert whether a screen reader is conveying the number as a word from the value text or as digits from `aria-valuenow`.
* E.g. is it saying "ten", or "10"?

CC @mcking65, @isaDC
